src/b3d/chisight/gen3d/model.py

```diff
@@ -123,19 +123,21 @@ def viz_trace(
 
     vertices = hyperparams["vertices"]
     b3d.rr_log_cloud(
-        vertices,
+        vertices[visibility_prob > 0.1],
```

Comment:
Change the viz: don't display the colors of points that are invisible (these can get out of sync and be confusing -- though maybe for some debugging having a way to view them will end up being useful).